libs/designsystem/src/lib/components/button/button.component.spec.ts

```diff
@@ -110,7 +128,7 @@ describe('ButtonComponent', () => {
 
     it('should render with correct color', () => {
       expect(element).toHaveComputedStyle({
-        color: getColor('white', 'contrast'),
```

Comment:
I agree that this assumption was faulty, and with your changes the tests are now actually testing the intended behavior.